packages/simplexmq/lib/simplexmq.dart

```diff
@@ -0,0 +1,8 @@
+/// Support for doing something awesome.
+///
+/// More dartdocs go here.
+library simplexmq;
+
+export "src/protocol.dart";
```

Comment:
You might not want to export all symbols (such as messageP, smpCommandP...), you can add show/hide to selectively show only some symbols or hide specific ones.

Reply:
I didn't figure out this syntax yet - can they be shown/hidden at the place of export? I thought it's at the place of import when I read the doc...

Also, is there a way to import the whole namespace as a name, to reference the symbols as Namespace.name?

Reply:
ExamYou can control in both import and export.

In export, to hide some sympols.
```yaml
// Export and hide some symbols
export "src/protocol.dart" hide smpCommandP, messageP;
```
Examples for import
```yaml
// Import and hide some symbols
import "package:simplexmq/simplexmq.dart" hide smpCommandP, messageP;

// Import only some symbols
import "package:simplexmq/simplexmq.dart" hide CmdErrorType;

// Import in a namespace
import "package:simplexmq/simplexmq.dart" as simplexmq;
```

Reply:
Cool.
Is there a way to list exported symbols inside protocol.dart itself?